src/inspector_js_api.cc

```diff
@@ -69,6 +69,7 @@ class JSBindingsConnection : public AsyncWrap {
                               JSBindingsConnection* connection)
                               : env_(env),
                                 connection_(connection) {
+      connection_->Detach();
```

Comment:
It’s not obviously correct to me that this is safe here. In fact, it seems explicitly _unsafe_, since the main reason to use a `BaseObjectPtr` member here would be to keep the connection object alive for the lifetime of the instance, which this PR undoes.